yaml/admin_areas.yaml

```diff
@@ -0,0 +1,9 @@
+filters:
+  - filter:
+      kind: admin_area
+      iso_code: true
+    min_zoom: 5
```

Comment:
Do we need to keep re-stating zoom 5?

Reply:
Unfortunately we do, because I'm shoe-horning the admin areas in through the same pipeline we use for features. This means the admin area features need a `min_zoom` and all the other trappings of "real" layer features, even though we're going to drop this layer and none of the features will ever appear in the output.

The alternative design would be to set up a parallel set of data structures for non-feature layers, but this seemed more invasive. I thought we might as well try the "pretend they're features" approach and see how that works - if we're happy enough with it, then it'll save us from doing a bunch more deep changes to how `tilequeue` works.